src/bmf.cpp

```diff
@@ -83,23 +87,25 @@ bool CxadbmfPlayer::xadplayer_load()
   size_t ptr = 0;
   int i;
 
-  if(xad.fmt != BMF)
+  if (xad.fmt != BMF)
     return false;
 
 #ifdef DEBUG
   AdPlug_LogWrite("\nbmf_load():\n\n");
 #endif
   if (tune_size < 6)
     return false;
-  if (!strncmp((char *)&tune[0],"BMF1.2",6))
+  if (!strncmp((char *)tune, "BMF1.2", 6))
```

Comment:
This one can sometimes be preferable the way it was, as it makes it a bit clearer you're talking about offset 0 in the file.

Reply:
It's a matter of taste, as I wrote above. While I don't like it very much, it's OK for me to restore the old `&[]` pattern if you prefer that. Just tell me whether you want me to replace the old commits or add a new one.
(Unrelated side note: I'd also prefer memcmp over strncmp here, but it's really minor and some people might not like it either.)

Reply:
I don't mind `strncmp` vs `memcmp` but since we're dealing with binary data here rather than text, `memcmp` seems a bit more logical.  It also gets rid of the typecast so happy with that change.